stixpy/coordinates/tests/test_frames.py

```diff
@@ -84,7 +89,9 @@ def test_stix_frame_map():
     data = np.random.rand(512, 512)
     obstime = "2023-01-01 12:00:00"
     solo = get_horizons_coord("solo", time=obstime)
-    coord = SkyCoord(0 * u.arcsec, 0 * u.arcsec, obstime=obstime, observer=solo, frame=STIXImaging)
+    coord = SkyCoord(
+        0 * u.arcsec, 0 * u.arcsec, obstime=obstime, obstime_end="2023-01-01 12:02:00", observer=solo, frame=STIXImaging
```

Comment:
Would it be better to define `obstime_end="2023-01-01 12:02:00"` below line 90 for consistency with `obstime`?